languagetool-language-modules/pt/src/test/java/org/languagetool/JLanguageToolTest.java

```diff
@@ -0,0 +1,52 @@
+/* LanguageTool, a natural language style checker 
+ * Copyright (C) 2005 Daniel Naber (http://www.danielnaber.de)
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+package org.languagetool;
+
+import org.junit.Test;
+import org.languagetool.language.PortugalPortuguese;
+import org.languagetool.language.AngolaPortuguese;
+import org.languagetool.language.BrazilianPortuguese;
+import org.languagetool.language.MozambiquePortuguese;
+
+import java.io.IOException;
+
+public class JLanguageToolTest {
+
+  @Test
+  public void testVariantsPortuguese() throws IOException {
+    Language lang = new PortugalPortuguese();
+    JLanguageTool tool = new JLanguageTool(lang);
+    tool.check("Isto é uma característica sua.");
```

Comment:
If the string is always the same, it should be moved into a variable/constant to avoid duplication.